src/providers/VideoSendingProvider.tsx

```diff
@@ -6,16 +6,14 @@ import React, {
   useState,
   createContext,
 } from "react";
-import {
-  DefaultPromisedWebSocketFactory,
-  ReconnectingPromisedWebSocket,
-  FullJitterBackoff,
-  DefaultDOMWebSocketFactory
-} from "amazon-chime-sdk-js";
 
+import { FullJitterBackoff } from "amazon-chime-sdk-js";
```

Comment:
Use a single quote for consistency (I see that the original file has a mix of single and double quotes.)

Reply:
Yes, have changed the other places in the file as well that used double quotes.